harvard-lts/jstor_aspace_oai_harvester
# Ss 257

**Change default harvest date a string.**
* * *

**JIRA Ticket**: [(link)](https://jira.huit.harvard.edu/browse/SS-257)

# What does this Pull Request do?
Makes default harvest date a string so it can be parsed to datetime by mongo write methods; before a parsing error was preventing mongo harvest records from getting written, this fixes that.
Changes status to "harvest" for harvester

# How should this be tested?

Deploy to dev (done)
Run ITest (done)
Run harvest with default date and check in mongo for results
mongo dev -> jstor_harvested_sumary -> {id:"20230413F"}

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
